### PR TITLE
Postman quickstart with collection and environment

### PR DESCRIPTION
**Pull request by SvenErikPy**
Added quick start ala vipps-login, with links to postman collection and environment.

image

### CODE REVIEW

**Comment by cloveras** [comment]
NIce! @rebekaburnett should review this, but it already looks great.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-<!-- START_METADATA
----
+## <!-- START_METADATA
```
Comment (rebekaburnett) [inline README.md:1]:
```suggestion
<!-- START_METADATA
```

File: vipps-checkout-api-quick-start.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<!-- START_METADATA
+
+title: Quick start
+sidebar_position: 60
```
Comment (rebekaburnett) [inline vipps-checkout-api-quick-start.md:4]:
```suggestion
sidebar_position: 5
```

**Review by rebekaburnett — APPROVED** [review]
This looks fantastic!! I fixed the metadata headers, which had gotten messed up - probably an the automatic linter.

**Review by rebekaburnett — APPROVED** [review]

**Pull request merged**